b2/sync/policy.py

```diff
@@ -170,8 +169,8 @@ class UpPolicy(AbstractFileSyncPolicy):
     def _make_transfer_action(self):
         return B2UploadAction(
             self._source_folder.make_full_path(self._source_file.name), self._source_file.name,
-            self._dest_folder.make_full_path(self._source_file.name),
-            self._get_source_mod_time(), self._source_file.latest_version().size
+            self._dest_folder.make_full_path(self._source_file.name), self._get_source_mod_time(),
+            self._source_file.latest_version().size
```

Comment:
this gets poorly formatted due to missing , at the end of last value, please correct it

Reply:
So we should always put a comma after the last argument to a function?  That seems awkward.

Reply:
The formatting is nicer.  My preference is actually to have one argument per line on multi-line function calls.